docs/changelog.rst

```diff
@@ -23,6 +27,7 @@ Changelog
 24.9.3
 ======
 - :ref:`ASYNC102 <async102>` and :ref:`ASYNC120 <async120>`:
+
   - handles nested cancel scopes
```

Comment:
unrelated fix: this was not getting parsed as a nested bullet list w/o an empty line